package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "endpoint_generator",
+  "version": "0.1.0",
+  "main": "index.js",
+  "repository": "git@git.endgames.local:ckim/endpoint_generator.git",
+  "author": "Chan Kim <ckim@endgame.com>",
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc --downlevelIteration",
```

Comment:
Not sure we need `--downlevelIteration` being that we're running with nodeJS 10. Unless its converting `import` into `require()` calls - then yes 😄